### DISCUSSION TITLE
De-mystifying ISCE Sign Convention

### DISCUSSION THREAD

**Discussion by elizabeth-ashley** [isce-framework/isce2#523] [Q&A]
In this notebook [isce2-docs](https://github.com/isce-framework/isce2-docs/tree/66a66ff31f4673eed9305f06982aaf4302a163cc)/[Notebooks](https://github.com/isce-framework/isce2-docs/tree/66a66ff31f4673eed9305f06982aaf4302a163cc/Notebooks)/[UNAVCO_2020](https://github.com/isce-framework/isce2-docs/tree/66a66ff31f4673eed9305f06982aaf4302a163cc/Notebooks/UNAVCO_2020)/[TOPS](https://github.com/isce-framework/isce2-docs/tree/66a66ff31f4673eed9305f06982aaf4302a163cc/Notebooks/UNAVCO_2020/TOPS)/[topsApp.ipynb](https://github.com/isce-framework/isce2-docs/blob/66a66ff31f4673eed9305f06982aaf4302a163cc/Notebooks/UNAVCO_2020/TOPS/topsApp.ipynb) it states:

![image](https://user-images.githubusercontent.com/63877993/178180594-f5cee0d6-411d-4f58-89a0-0d625625b9aa.png)

However, I just processed data for the Ridgecrest 2019 earthquake using this data (using snaphu_mcf):
Sentinel-1 Ascending track T64: reference = 20190622 & secondary = 20190716 and had to multiply my data by a negative to get the correct sign convention. [Click here to see my reference to the processed data for this earthquake.](https://topex.ucsd.edu/SV_7.1/index.html#:~:text=1%20Ridgecrest%20earthquake%20struck%20on,the%20city%20of%20Ridgecrest%2C%20CA.)

**Image 1 produced from:** 
`gdal_calc.py -A filt_topophase.unw.geo.vrt --A_band=2 --calc="A*1*0.05546576/12.5663706" --outfile=1test_filt_topophase.unw_m.geo.tif  --format=GTiff --NoDataValue=-9999 --overwrite`
![image](https://user-images.githubusercontent.com/63877993/178181174-2d66e277-eb30-40e6-b1d1-2953a8d0c3e5.png)

**Image 2 produced from (_multiplying by negative_):** 
`gdal_calc.py -A filt_topophase.unw.geo.vrt --A_band=2 --calc="A*-1*0.05546576/12.5663706" --outfile=2test_filt_topophase.unw_m.geo.tif  --format=GTiff --NoDataValue=-9999 --overwrite`
![image](https://user-images.githubusercontent.com/63877993/178181192-9128fa22-3c91-4681-ae4b-b6648d3280e2.png)

**My understanding is this:** In the example case for ground moving away from the satellite the phase_reference would be smaller than the phase_secondary meaning that the SAR sensor was closer at the reference time and further away in the secondary time. In method 1,  the phase change value would be negative meaning that the ground moving away are negative values. In method 2, the phase change value would be positive meaning that the ground moving away are positive values. Same logic would apply if the case had a phase_reference larger than phase_secondary. 
![image](https://user-images.githubusercontent.com/63877993/178181889-e526b218-9c07-40c5-9972-2abcc2af1b01.png)

**QUESTION: So if method 1 is the case for the ISCE output when using a secondary acquisition date that comes after the reference date (as stated above), why did I need to treat it as if it was calculating using method 2? Is it because the statement was for a Descending track?**

**Comment by scottstanie** [comment]
Hi, the general convention for relating phase to range is  $\phi_k = -4 \pi r_k$. This means you are correct that (using method 1) 
$$\Delta \phi_{1,2} = \phi_1 - \phi_2 = -4 \pi (r_1 - r_2) = 4 \pi (r_2 - r_1)$$

i.e. *increase* in the line of sight distance from time 1 to 2 leads to a *positive* phase. as you found in your interferogram. 

But the ISCE line of sight maps in the "geometry" folders all have the convention that the LOS vector points *from* ground *to* satellite, so often people will plot the LOS displacement so that a negative value means motion away from the satellite. The MintPy FAQ also mentions this https://github.com/insarlab/MintPy/blob/f390551ae4c9a445c9fde9219d0ee5984a8a22fb/docs/FAQs.md#1-whats-the-sign-convention-of-the-line-of-sight-data

I believe the reason for having this convention is so that getting a positive number for LOS displacement in both ascending and descending maps leads to a positive number in the final vertical displacement map that people often care about.
